ci-operator/step-registry/baremetal/lab/agent/day2/baremetal-lab-agent-day2-commands.sh

```diff
@@ -0,0 +1,193 @@
+  # shellcheck disable=SC2154
+  for bmhost in $(yq e -o=j -I=0 '.[] | select(.name|test("-a-"))' "${SHARED_DIR}/hosts.yaml"); do
+   # shellcheck disable=SC1090
+   . <(echo "$bmhost" | yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")')
+   if [ "${transfer_protocol_type}" == "cifs" ]; then
+     IP_ADDRESS="$(dig +short "${AUX_HOST}")"
+     iso_path="${IP_ADDRESS}/isos/${CLUSTER_NAME}.node.iso"
+   else
+     # Assuming HTTP or HTTPS
+     iso_path="${transfer_protocol_type:-http}://${AUX_HOST}/${CLUSTER_NAME}.node.iso"
+   fi
+   mount_virtual_media "${host}" "${iso_path}"
+  done
+
+  wait
+  if [ -f /tmp/virtual_media_mount_failed ]; then
+   echo "Failed to mount the ISO image in one or more hosts"
+   exit 1
+  fi
+;;
+"pxe")
+  ### Create pxe files
+  echo -e "\nCreating PXE files..."
+  oinst agent create pxe-files
+  ### Copy the image to the auxiliary host
+  echo -e "\nCopying the PXE files into the bastion host..."
+  scp "${SSHOPTS[@]}" "${DAY2_INSTALL_DIR}"/boot-artifacts/agent.*-vmlinuz* \
+   "root@${AUX_HOST}:/opt/dnsmasq/tftpboot/${CLUSTER_NAME}/vmlinuz_${gnu_arch}"
+  scp "${SSHOPTS[@]}" "${DAY2_INSTALL_DIR}"/boot-artifacts/agent.*-initrd* \
+   "root@${AUX_HOST}:/opt/dnsmasq/tftpboot/${CLUSTER_NAME}/initramfs_${gnu_arch}.img"
+  scp "${SSHOPTS[@]}" "${DAY2_INSTALL_DIR}"/boot-artifacts/agent.*-rootfs* \
+   "root@${AUX_HOST}:/opt/html/${CLUSTER_NAME}/rootfs-${gnu_arch}.img"
+;;
+*)
+  echo "Unknown install mode: ${BOOT_MODE}"
+  exit 1
+esac
+
+day2_IPs=""
+
+# shellcheck disable=SC2154
+for bmhost in $(yq e -o=j -I=0 '.[] | select(.name|test("-a-"))' "${SHARED_DIR}/hosts.yaml"); do
+  # shellcheck disable=SC1090
+  . <(echo "$bmhost" | yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")')
+  day2_IPs+="${ip},"
+  echo "Power on #${host} (${name})..."
+  timeout -s 9 10m ssh "${SSHOPTS[@]}" "root@${AUX_HOST}" prepare_host_for_boot "${host}" "${BOOT_MODE}"
+done
+
+day2_IPs=${day2_IPs%,}
+touch /tmp/output.txt
+
+echo "Monitoring day2 workers and pending CSRs..."
+
+/cli/oc adm node-image monitor --ip-addresses "${day2_IPs}" -a "${day2_pull_secret}" --insecure=true 2>&1 | \
+  tee /tmp/output.txt | while IFS= read -r line; do
+  echo "$line"
+  if [[ "$line" = *"with signerName kubernetes.io/kube-apiserver-client-kubelet and username system:serviceaccount:openshift-machine-config-operator:node-bootstrapper is Pending and awaiting approval"* ]]; then
+    node_ip=$(echo "$line" | sed 's/^.*Node \(.*\): CSR.*$/\1/')
+    csr=$(echo "$line" | sed 's/^.*CSR \([^ ]*\).*$/\1/')
+    echo "Approving CSR $csr for node $node_ip"
+    oc adm certificate approve "$csr"
+  fi
+  if [[ "$line" = *"with signerName kubernetes.io/kubelet-serving and username "*" is Pending and awaiting approval"* ]]; then
+    node_ip=$(echo "$line" | sed 's/^.*Node \(.*\): CSR.*$/\1/')
+    csr=$(echo "$line" | sed 's/^.*CSR \([^ ]*\).*$/\1/')
+    echo "Approving CSR $csr for node $node_ip"
+    oc adm certificate approve "$csr"
+  fi
```

Comment:
```suggestion
  if [[ "$line" = *"with signerName kubernetes.io/kube-apiserver-client-kubelet and username system:serviceaccount:openshift-machine-config-operator:node-bootstrapper is Pending and awaiting approval"* ]] || [[ "$line" = *"with signerName kubernetes.io/kubelet-serving and username "*" is Pending and awaiting approval"* ]]; then
    node_ip=$(echo "$line" | sed 's/^.*Node \(.*\): CSR.*$/\1/')
    csr=$(echo "$line" | sed 's/^.*CSR \([^ ]*\).*$/\1/')
    echo "Approving CSR $csr for node $node_ip"
    oc adm certificate approve "$csr"
  fi
```

Aren't the two blocks the same?